cassandra-2.2/src/main/java/org/apache/cassandra/auth/Cassandra22SystemAuthRoles.java

```diff
@@ -60,13 +64,28 @@ public boolean roleMissing(String dn)
         return rows.result.isEmpty();
     }
 
-    public void createRole(String roleName, boolean superUser)
+    public void createRole(String roleName, boolean superUser, Optional<String> defaultRoleMembership)
     {
         final CreateRoleStatement createStmt = (CreateRoleStatement) QueryProcessor.getStatement(format(CREATE_ROLE_STATEMENT_WITH_LOGIN, roleName, superUser),
                                                                                                  getClientState()).statement;
 
         createStmt.execute(new QueryState(getClientState()),
                            QueryOptions.forInternalCalls(LOCAL_ONE, singletonList(ByteBufferUtil.bytes(roleName))));
+
+        if (defaultRoleMembership.isPresent()) {
```

Comment:
could you please be consistent with code style and place braces on new line? Feel free to fix this all class where braces are not done line that.